### PR TITLE
Fix dropdowns in new WI

### PR DESCRIPTION
**Pull request by michaelkleinhenz**
This fixes the empty dropdowns for the new WI dialog. The issue was caused by the dropdown values changing after the dialog is shown. The typeahead dropdown did not support Input() changes. It does support this now.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

**Comment by sanbornsen** [comment]
Everything looks good to me but, don't you think the default are and iteration should be selected by default?
screen shot 2017-04-22 at 12 45 49 pm
Also if I select an area it stays None. Works fine with iteration.

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/work-item/work-item-detail/work-item-detail.component.ts [hunk 1]
```diff
@@ -156,19 +156,19 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
             if (this.headerEditable && typeof(this.title) !== 'undefined') {
               this.title.nativeElement.focus();
             }
-          }
```
Comment (sanbornsen) [inline src/app/work-item/work-item-detail/work-item-detail.component.ts]: There was one PR (https://github.com/fabric8io/fabric8-planner/pull/1733) got merged to master that will fix this part.

**Comment by michaelkleinhenz** [comment]
@sanbornsen there should no "None" in the boxes anymore. Where do you see that?

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

**Comment by michaelkleinhenz** [comment]
@nimishamukherjee @sanbornsen Ah, I get it. No, that is actually an expected behaviour. The "None" only appears on "old" work items that don't have an area set. With the old core behaviour, new WIs were created without an area. With the new core, the default/root area is always set. The "None" only appears (and should appear) on "incorrect" WIs not having an area. Can you confirm that it is working that way?

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1723-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**